svm/src/program_loader.rs

```diff
@@ -217,21 +217,6 @@ pub fn load_program_with_pubkey<CB: TransactionProcessingCallback, FG: ForkGraph
 
     let mut timings = ExecuteDetailsTimings::default();
     load_program_metrics.submit_datapoint(&mut timings);
-    if !Arc::ptr_eq(
```

Comment:
This branch was only taken during a feature set change which affects RBPF. Thus, removing it has no affect on the normal operation outside the epoch boundary recompilation phase.